### ISSUE TITLE
WebClient Failed to execute 'createObjectURL' on 'URL'

### ISSUE THREAD

**Issue by shutupzk** [3DStreamingToolkit/3DStreamingToolkit#310]
app.js:139 Uncaught TypeError: Failed to execute 'createObjectURL' on 'URL': No function was found that matched the signature provided.
    at HTMLButtonElement.onRemoteStreamAdded (app.js:139)

**Comment by Ziriax** [comment]
It seems setting the `src` to ` MediaStream` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/srcObject)

I fixed this by changing the `app.js` code like this:

```
 function onRemoteStreamAdded(event) {
    document.getElementById('renderers').style.display = 'none';
    remoteVideoElement = document.getElementById('remote-video');
    console.log('Adding remote stream...');
    try {
      remoteVideoElement.srcObject = event.stream;
    } catch (error) {
      remoteVideoElement.src = URL.createObjectURL(event.stream);
    }    
    console.log('Remote stream added');
    remoteVideoElement.play();
  }
```

**Comment by anderm** [comment]
@Ziriax Thank you for your reply. Indeed, the srcObject was deprecated, we would love to receive a PR based on your fix!

**Comment by anderm** [comment]
Closed with #315

**Comment by L-Naej** [comment]
Warning: the website doc still points to an old version of the web client with this bug not corrected!
